<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainCameraWgt</class>
 <widget class="QWidget" name="MainCameraWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="cameraTitle" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="CVLayout" stretch="1,1,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="TGFrame" name="modelWgt" native="true"/>
    </item>
    <item>
     <widget class="TGFrame" name="pingWgt" native="true"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5" rowspan="2">
       <widget class="SteerWheel" name="wheelWgt" native="true"/>
      </item>
      <item row="2" column="4" colspan="3">
       <widget class="GearWgt" name="gearWgt" native="true"/>
      </item>
      <item row="1" column="7" rowspan="2">
       <widget class="TriangleFill" name="throttleWgt" native="true"/>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="TriangleFill" name="brakeWgt" native="true"/>
      </item>
      <item row="1" column="6">
       <widget class="RingDial" name="tachometerWgt" native="true"/>
      </item>
      <item row="1" column="4">
       <widget class="RingDial" name="speedWgt" native="true"/>
      </item>
      <item row="0" column="8" rowspan="3" colspan="2">
       <widget class="QWidget" name="zero_2" native="true"/>
      </item>
      <item row="0" column="0" rowspan="2" colspan="2">
       <widget class="TGFrame" name="midVideoFrame1" native="true">
        <layout class="QHBoxLayout" name="minPLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="OpenGLWidget" name="backPlayWgt" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="backTrackLine" name="widget_ReversingTrackLine_b" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="OpenGLWidget" name="frontPlayWgt" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="backTrackLine" name="widget_ReversingTrackLine_f" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="ReversingTrackLine" name="widget_ReversingTrackLine_old" native="true">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TGFrame</class>
   <extends>QWidget</extends>
   <header>Component/TGFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QWidget</extends>
   <header>Component/OpenglWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RingDial</class>
   <extends>QWidget</extends>
   <header>Component/RingDial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TriangleFill</class>
   <extends>QWidget</extends>
   <header>Component/TriangleFill.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GearWgt</class>
   <extends>QWidget</extends>
   <header>Component/GearWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SteerWheel</class>
   <extends>QWidget</extends>
   <header>Component/SteerWheel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReversingTrackLine</class>
   <extends>QWidget</extends>
   <header>Component/ReversingTrackLine.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>backTrackLine</class>
   <extends>QWidget</extends>
   <header>Component/backTrackLine.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
